### DISCUSSION TITLE
v2.15.5

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#459] [General]
Fixes.

**Announcement**: Support for Android 5-8.1 is ending with next major release. See #452 for more information.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.15.4...v2.15.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.5'>v2.15.5</a>.</em>

**Comment by l2ol3-l2ol3** [comment]
Hello,
Do you know if it is possible to enable tethering at startup of your app?
I use MacroDroid to automate the activation of the hotspot, and the launch of your application but I still need to manually activate ap0.

Good day =)

Rob

**Reply by Mygod** [reply]
Use the monitor feature at the top right corner of the app.

**Comment by l2ol3-l2ol3** [comment]
Oh...My god. ( 🤦 )

Simple, effective.
I did not understand that this option was used for this.
Thank à lot.
Good job.


...

________________________________
